ui/components/VerticalTabs.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <div
+    class="vertical-tabs"
+    :class="computedRootClasses"
+  >
+    <div
+      v-for="tab in computedTabs"
+      :key="tab.key"
+      class="vertical-tabs__tab"
+      :class="tab.classes"
+      @click="onTabClick(tab)"
+    >
+      <div
+        v-if="props.linear"
+        class="vertical-tabs__tab-index"
+      >
+        <ui-icon
+          v-if="tab.skipStep"
+          icon-name="googleArrowDownwardBaseline"
+          :color="COLORS_DICT.NICE_BLUE"
+          size="18"
+        />
+        <span v-else-if="tab.inactive || tab.active">{{ tab.step }}</span>
+        <ui-icon
+          v-else
+          icon-name="googleCheckBaseline"
+          :color="COLORS_DICT.NICE_BLUE"
+          size="18"
+        />
+      </div>
+      <p>{{ tab.label }}</p>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { computed } from 'vue';
+
+import { COLORS_DICT } from '~/constants/colors';
+
+const props = defineProps({
+  tabs: {
+    type: Array,
+    required: true,
+  },
+  linear: {
+    type: Boolean,
+    default: false,
+  },
+});
+
+const activeTabKey = defineModel('activeTabKey', {
+  type: String,
+  default: '',
+});
+
+const activeTabIndex = computed(() =>
+  props.tabs.findIndex((tab) => tab.key === activeTabKey.value),
+);
+
+const computedTabs = computed(() =>
+  props.tabs.map((tab, index) => ({
+    ...tab,
+    step: index + 1,
+    active: index === activeTabIndex.value,
+    inactive: props.linear && index > activeTabIndex.value,
```

Comment:
I would love parenthesis here :-)

Reply:
They are removed automatically by Prettier since they are not "required" 🤔. I checked the configuration options and there is no option to change it.